corva/models/stream.py

```diff
@@ -29,38 +46,24 @@ class AppMetadata(CorvaBaseModel):
 
 class StreamEventMetadata(CorvaBaseModel):
     app_stream_id: int
-    source_type: Optional[str] = None
     apps: Dict[str, AppMetadata]
 
+    source_type: Optional[
+        Literal['drilling', 'drillout', 'frac', 'wireline']
+    ] = pydantic.Field(None, description='Source Data Type')
+    log_type: Optional[Literal['time', 'depth']] = pydantic.Field(
+        None, description='Source Log Type'
+    )
+    log_identifier: Optional[str] = pydantic.Field(
+        None, description='Unique log identifier, only available on depth based streams'
+    )
 
-class StreamEventData(CorvaBaseModel):
-    app_key: Optional[str] = None
+
+class StreamEvent(BaseEvent):
+    asset_id: int
```

Comment:
Same comment here: can't these fields move to the BaseEvent?

Reply:
https://github.com/corva-ai/python-sdk/pull/25#discussion_r573624389